# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1320782128 28800
#      Tue Nov 08 11:55:28 2011 -0800
# Node ID 35f59bd285bfebae2d4e7be2502acca97c3b1723
# Parent  e48529b3dbd49b7e0c7e10d91b484e0e23c9caaf
doc: Change bitunpack docstring to refer to itself, not bitpack

* typecast.cc (bitunpack): Change docstring to refer to bitunpack, not bitpack

diff --git a/src/DLD-FUNCTIONS/typecast.cc b/src/DLD-FUNCTIONS/typecast.cc
--- a/src/DLD-FUNCTIONS/typecast.cc
+++ b/src/DLD-FUNCTIONS/typecast.cc
@@ -377,17 +377,17 @@ do_bitunpack (const ArrayType& array)
       bits += CHAR_BIT;
     }
 
   return retval;
 }
 
 DEFUN_DLD (bitunpack, args, ,
   "-*- texinfo -*-\n\
-@deftypefn {Loadable Function} {@var{y} =} bitpack (@var{x})\n\
+@deftypefn {Loadable Function} {@var{y} =} bitunpack (@var{x})\n\
 Return an array @var{y} corresponding to the raw bit patterns of\n\
 @var{x}.  @var{x} must belong to one of the built-in numeric classes:\n\
 \n\
 @example\n\
 @group\n\
   \"char\"\n\
   \"int8\"\n\
   \"int16\"\n\
